tools/go-cli/build.gradle

```diff
@@ -0,0 +1,60 @@
+ext.dockerImageName = "whisk/go-cli"
+ext.dockerContainerName = "go-cli"
+apply from: "../../docker.gradle"
+
+ant.condition(property: "family", value: "linux") {
+    os(family: "unix")
+}
+
+ant.condition(property: "family", value: "mac") {
+    os(family: "mac")
+}
+
+ant.condition(property: "arch", value: "amd64") {
+    os(arch: "amd64")
+}
+
+ant.condition(property: "arch", value: "amd64") {
+    os(arch: "x86_64")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "386")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "i386")
+}
+
+ant.condition(property: "arch", value: "386") {
+    os(arch: "x86_32")
+}
+
+task removeBinary(type: Delete) {
+    delete "${projectDir}/../../bin/go-cli"
+}
+
+task distBinary(dependsOn: [removeBinary, distDocker]) << {
+    run(dockerBinary + ["rm", "-f", dockerContainerName], true)
+    run(["mkdir", "${projectDir}/../../bin/go-cli/"])
```

Comment:
Well, yeah, but it also puts the outcome of the `distTar` task in `build/distributions` which is idiomatic for gradle. From there on, we can continue with copying the stuff around to wherever we need it, keeping the `distBinary`  task itself gradle idiomatic.